[CAW-365] chore: update docs (#37)
GetUnblock/sdk-javascript: docs/AUTH.md

    prod: false, // If true, Production environment will be used otherwise Sandbox will be used instead
  });

  // SDK API call example
  const result = await sdk.auth.emailSession({
    userUuid: '[an user uuid value]',
    code: '[The code received in the email]',
  });
})();
```

###### JavaScript

```javascript
const getunblockSDK = require('@getunblock/sdk').default;
const { AuthenticationMethod } = require('@getunblock/sdk');

(async () => {
  // setup SDK
  const sdk = getunblockSDK({
    apiKey: 'API-Key [Some merchant Key]', // Key generated at the moment the merchant was created in getunblock system
    prod: false, // If true, Production environment will be used otherwise Sandbox will be used instead
  });

  // SDK API call example
  const result = await sdk.auth.emailSession({
    userUuid: '[an user uuid value]',
    code: '[The code received in the email]',
  });
})();
```

#### createSiweMessage

Will generate Siwe login message.

<div><pre>createSiweMessage(address: string, statement: string, domain: string, chainId: number): string</pre></div>

##### Overview

> Will generate Siwe login message and signature, then use it to login with SIWE. Returns user uuid and unblock session id.

Reviewer: ```suggestion
Will generate a Siwe login message that is required for login with SIWE.
```

      "API-Key [Some merchant Key]", // Key generated at the moment the merchant was created in getunblock system
    prod: false, // If true, Production environment will be used otherwise Sandbox will be used instead
  });

  // Ideally it must be an ethereum provider signer implementation
  const yourOwnProviderSigner = YouOwnProviderSignerFactory.get();

  // SDK API call example
  const result = await sdk.auth.generateSiweLoginMessage(
      providerSigner: yourOwnProviderSigner,
      chainId: '8001',
      signingUrl: string,
  );
})();
```

#### walletSiweLogin

<div><pre>walletSiweLogin(params: <a href="#WalletSiweLoginRequest">WalletSiweLoginRequest</a>): Promise&#60;<a href="#WalletSiweLoginResponse">WalletSiweLoginResponse</a>&#62;</pre></div>

##### Overview

> Will generate Siwe login message and signature, then use it to login with SIWE. Returns user uuid and unblock session id.

Reviewer: ```suggestion
Will generate a Siwe login message and signature then it logs in with SIWE. Returns user uuid and unblock session id.
```
